packages/utils/odsp-doclib-utils/src/odspErrorUtils.ts

```diff
@@ -148,6 +148,16 @@ export function createOdspNetworkError(
             }
             break;
         case 404:
+            if (parseResult.success) {
+                const responseError = parseResult.errorResponse.error;
+                const redirectLocation = responseError["@error.redirectLocation"];
+                if (redirectLocation !== undefined) {
+                    const propsWithRedirectLocation = { ...driverProps, newSiteUri: redirectLocation };
```

Comment:
Is this new site location, or new file (item) location?

Reply:
I think it is newSite location, but I will confirm.

Reply:
It is new complete redirect location to which call should be made. So renaming it as such.